doc_source/nvme-ebs-volumes.md

```diff
@@ -91,6 +91,19 @@ Volume ID: vol-01324f611e2463981
 
 Amazon Linux also creates a symbolic link from the device name in the block device mapping \(for example, `/dev/sdf`\), to the NVMe device name\.
 
+**FreeBSD AMIs**  
```

Comment:
<3 FreeBSD and thank you for maintaining it! This paragraph looks good to me, but I do think it should come after the  "other linux AMIs" section, based purely on frequency of usage.

Reply:
Yeah, I wondered about that.  In the end I figured that it made more sense to keep the platforms which use `ebsnvme-id` together; but I'm ok with moving this if Amazon would prefer to put FreeBSD at the end.

(Of course, if you want to go *completely* based on frequency of usage, I think "Other Linux" would come before "Amazon Linux"...)

Reply:
#114